ParcelInvoiceMaker/ParcelInvoiceMaker/BusinessLogic.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  BusinessLogic.swift
+//  ParcelInvoiceMaker
+//
+//  Created by MIN SEONG KIM on 2024/01/25.
+//
+
+import Foundation
+
+struct BusinessLogic {
+    // 주소 유효성 체크
+    static func validateAddress(_ address: String) -> Bool {
```

Comment:
요건 사실 취향이긴 한데, 어딘가에서 와일드 카드를 지양하는게 좋다는 걸 본거같아서 suggestion 드립니당
```suggestion
    static func validate(address: String) -> Bool {
```